### PR TITLE
build: remove obscure versions

### PR DESCRIPTION
**Pull request by Yeti-or**
:baby_chick: Download canary assets:

[colors--canary.351.4193555506.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/colors--canary.351.4193555506.xml)  
[color_metro_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_ios-swift--canary.351.4193555506.swift)  
[color_metro_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_kotlin--canary.351.4193555506.kt)  
[color_metro_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_metro_react-native--canary.351.4193555506.ts)  
[color_sbermarket_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.351.4193555506.swift)  
[color_sbermarket_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.351.4193555506.kt)  
[color_sbermarket_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.351.4193555506.ts)  
[color_sberprime_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.351.4193555506.swift)  
[color_sberprime_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.351.4193555506.kt)  
[color_sberprime_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.351.4193555506.ts)  
[color_selgros_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_ios-swift--canary.351.4193555506.swift)  
[color_selgros_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_kotlin--canary.351.4193555506.kt)  
[color_selgros_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_selgros_react-native--canary.351.4193555506.ts)  
[color_smbusiness_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_ios-swift--canary.351.4193555506.swift)  
[color_smbusiness_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_kotlin--canary.351.4193555506.kt)  
[color_smbusiness_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_smbusiness_react-native--canary.351.4193555506.ts)  
[PlasmaTokensColor--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/PlasmaTokensColor--canary.351.4193555506.swift)  
[shadow_sbermarket_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.351.4193555506.ts)  
[typo_mage_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.351.4193555506.swift)  
[typo_mage_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.351.4193555506.kt)  
[typo_mage_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.351.4193555506.ts)  
[typo_plasma_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.351.4193555506.swift)  
[typo_plasma_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.351.4193555506.kt)  
[typo_plasma_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.351.4193555506.ts)  
[typo_ruler_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.351.4193555506.swift)  
[typo_ruler_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.351.4193555506.kt)  
[typo_ruler_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.351.4193555506.ts)  
[typo_sage_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.351.4193555506.swift)  
[typo_sage_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.351.4193555506.kt)  
[typo_sage_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.351.4193555506.ts)  
[typo_sbermarket_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.351.4193555506.swift)  
[typo_sbermarket_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.351.4193555506.kt)  
[typo_sbermarket_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.351.4193555506.ts)  
[typo_soulmate_ios-swift--canary.351.4193555506.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.351.4193555506.swift)  
[typo_soulmate_kotlin--canary.351.4193555506.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.351.4193555506.kt)  
[typo_soulmate_react-native--canary.351.4193555506.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.351.4193555506.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.141.1-canary.351.4193555506.0
  npm install @salutejs/plasma-core@1.97.1-canary.351.4193555506.0
  npm install @salutejs/plasma-hope@0.1.1-canary.351.4193555506.0
  npm install @salutejs/plasma-icons@1.125.1-canary.351.4193555506.0
  npm install @salutejs/plasma-temple@1.136.1-canary.351.4193555506.0
  npm install @salutejs/plasma-ui@1.169.1-canary.351.4193555506.0
  npm install @salutejs/plasma-web@1.167.1-canary.351.4193555506.0
  npm install @salutejs/plasma-sb-utils@0.95.1-canary.351.4193555506.0
  # or 
  yarn add @salutejs/plasma-b2c@1.141.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-core@1.97.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-hope@0.1.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-icons@1.125.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-temple@1.136.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-ui@1.169.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-web@1.167.1-canary.351.4193555506.0
  yarn add @salutejs/plasma-sb-utils@0.95.1-canary.351.4193555506.0
  ```
</details>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -7265,34 +7265,7 @@
     },
     "@salutejs/plasma-cy-utils": {
       "version": "file:utils/plasma-cy-utils",
-      "dev": true,
-      "dependencies": {
-        "@salutejs/plasma-tokens": {
-          "version": "1.17.0",
```
Comment (Yeti-or) [inline package-lock.json:7271]: таких версий просто не существует:

```sh
npm info @salutejs/plasma-tokens

[
  '1.17.1-canary.1.6c1bd60c24868a33f35c1eb64cb2435fa83702e8.0',
  '1.17.1-canary.1.aa27d430fdf4866b1c825b7e8525cfb31fdece89.0',
  '1.17.1-canary.1.afffff869173c639b2fa9e662378dde636344625.0',
  '1.17.1-canary.1.b09d29c80045d5ca030bb956940824b2889b9d25.0',
  '1.18.0',
  '1.18.1-canary.17.2277171592.0',
  '1.18.1',
  '1.19.0-canary.115.2782398305.0',
  '1.19.0-canary.115.2789919209.0',
  '1.19.0-canary.115.2790340617.0',
  '1.19.0-canary.119.2803560557.0',
  '1.19.0-canary.119.2831025595.0',
  '1.19.0-canary.121.2816426479.0',
  '1.19.0-canary.121.2831035764.0',
  '1.19.0-canary.122.2823454341.0',
  '1.19.0',

...
```

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-351/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out comparison report in job artifacts!

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-351/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-351/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-351/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-351/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-351/temple-storybook/

**Pull request merged**